src/components/bcros/filing/item/ContinuationOut.vue

```diff
@@ -13,7 +13,7 @@
           <strong>
             {{ $t('text.filing.continuation.continuedOutOn') }} {{ continuationOutDate }},
             {{ $t('text.general.to') }}
-            {{ foreignJurisdiction }} {{ $t('text.filing.continuation.underTheName') }} {{ currentBusinessName }}.
+            {{ foreignJurisdiction }} {{ $t('text.filing.continuation.underTheName') }} {{ continuationOutLegalName }}.
```

Comment:
Before:
![image](https://github.com/user-attachments/assets/037f33f4-79dd-41c2-82b3-47298314351e)

After:
![image](https://github.com/user-attachments/assets/17b450d2-8563-4544-8507-209cfad12fb5)


(highlight done for screenshot purposes only)